Do not fail the wxPython build when using a wxWidgets-3.2.5 base.

Generated by:
git diff 5622abb73deaa26dc2f6dc4cd8b4b2050396b49a..78938da1218483024b3a7acf55b5fb5513882916 etg/


diff --git etg/window.py etg/window.py
index c388df76..0c035f14 100644
--- a/etg/window.py
+++ a/etg/window.py
@@ -251,6 +251,23 @@ def run():
             return NULL;
         #endif
         """)
+    c.find('GetOrCreateAccessible').setCppCode("""\
+        #if wxUSE_ACCESSIBILITY
+            return self->GetOrCreateAccessible();
+        #else
+            wxPyRaiseNotImplemented();
+            return NULL;
+        #endif
+        """)
+    c.find('CreateAccessible').factory = True
+    c.find('CreateAccessible').setCppCode("""\
+        #if wxUSE_ACCESSIBILITY
+            return self->CreateAccessible();
+        #else
+            wxPyRaiseNotImplemented();
+            return NULL;
+        #endif
+        """)
     c.find('SetAccessible.accessible').transfer = True
     c.find('SetAccessible').setCppCode("""\
         #if wxUSE_ACCESSIBILITY
